jspwiki-util/src/main/java/org/apache/wiki/util/FileUtil.java

```diff
@@ -108,8 +108,8 @@ public static String runSimpleCommand( final String command, final String direct
         final StringBuilder result = new StringBuilder();
         final Process process = Runtime.getRuntime().exec( command, null, new File( directory ) );
 
-        try( final BufferedReader stdout = new BufferedReader( new InputStreamReader( process.getInputStream() ) );
-             final BufferedReader stderr = new BufferedReader( new InputStreamReader( process.getErrorStream() ) ) ) {
+        try( final BufferedReader stdout = new BufferedReader( new InputStreamReader( process.getInputStream(), StandardCharsets.UTF_8 ) );
```

Comment:
the only method using this one is expecting it to be on ISO-8859. Perhaps passing as an argument `engine.getContentEncoding()` and use it, ensuring it's used throughout the entire call, would be better?

Reply:
changed